docs/0.5/concepts/security_considerations.md

```diff
@@ -153,9 +153,9 @@ Guide](https://sawtooth.hyperledger.org/docs/sabre/nightly/master/application_de
 
 ## Ports and REST API Connections
 
-By default, Splinter uses ports **8084** and **8085** for communication.
+By default, Splinter uses ports **8080** and **8084** for communication.
```

Comment:
This should be 8044 instead of 8084.  Also, it's missing a reference to 8043, the service port

Reply:
Updated 8084 -> 8044